test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -1385,7 +1455,6 @@ await CliTester.Run(new RunConfiguration[]
                        $"start --port {_funcHostPort}"
                    },
                    ExpectExit = true,
-                   ExitInError = true,
```

Comment:
exitInError does not work when we have to assert the exit error, so I removed this since it was not relevant anymore